### ISSUE TITLE
Add note to backup selection indicating the consequences of ignoring it

### ISSUE THREAD

**Issue by gary-rowe** [bitcoin-solutions/multibit-hd#510]
User feedback from @crwatkins has indicated that a better description of both the purpose of the backup directory and the consequences of ignoring it would be helpful. 

It is not obvious from the choice of "cloud backup" that the backup should be on a separate machine and is strongly encrypted to make this choice safe so this should be stated explicitly.

**Comment by jim618** [comment]
We already mention in the 'steps to create a wallet' what this directory is for.

General users aren't going to understand 'strongly encrypted' so there is little point in mentioning it on the UI. It's just techno-babble. It should be in the Help though what the algorithm is.

We can add in a sentence on the cloud backup directory to explain normatively what this directory is.
We shouldn't frighten users if they don't set this - it would be better just to nag them to set it - which is already an issue.

**Comment by crwatkins** [comment]
My original comments related to concerns about implying that this directory must be stored in the cloud,
and what a user would do here if they didn't have any "cloud directories" (fairly common?).

I also mentioned that I wouldn't even store an encrypted wallet on all the providers mentioned, which probably prompted Gary's "strongly encrypted" note.

**Comment by jim618** [comment]
You are considerably more technically able than most MultiBit users.
We can say in the help what the backups are (we already describe the schedule) but it;s just confusing to say on the UI 'encrypted with an AES key derived from a combination of trapdoor functions based on the mnemonic seed phrase' or whatever. It's just gobble-de-gook to most people.

**Comment by crwatkins** [comment]
Sorry, I think I've drawn this totally away from my original point, which was:

Wallet:  "Enter your cloud backup directory" (or something like that)
New user:  "I don't have a cloud backup directory.  What do I do now?"

**Comment by jim618** [comment]
Agreed we should make it clear that you don't need to set up a cloud backup dir and can just click next.

Then, later, we'll nag them if they haven't set it (probably with a link to the Help to explain more).
